barretenberg/cpp/src/barretenberg/commitment_schemes/ipa/ipa.hpp

```diff
@@ -183,10 +183,12 @@ template <typename Curve> class IPA {
      */
     static bool verify(const std::shared_ptr<VK>& vk,
```

Comment:
Will we ever need a recursive verify for IPA? If so, this would require changes since it just treats the output of receive_from_prover as a native type. It would throw a compile error if we ever tried to instantiate IPA with a stdlib curve.